### PR TITLE
Project cleanup

### PR DESCRIPTION
**Pull request by jagoda**

### CODE REVIEW

**Comment by jagoda** [comment]
If this is accepted we should cut a new release (by pushing a new tag) and grant access to @lifeomic/open-source.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (jagoda) [inline LICENSE]: This file gets us the GitHub license detection.

File: package.json [hunk 2]
```diff
@@ -53,5 +53,8 @@
     "aws-xray-sdk-core": "^1.2.0",
     "graphql-middleware": "^1.2.5",
     "is-promise": "^2.1.0"
+  },
+  "publishConfig": {
+    "access": "public"
```
Comment (jagoda) [inline package.json]: This marks the package as public even though it is scoped.

File: .travis.yml [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
 after_success: yarn coverage
-
 notifications:
   email:
     on_success: never
     on_failure: always
+deploy:
+  provider: npm
+  email: $NPM_EMAIL
+  api_key: $NPM_KEY
```
Comment (jagoda) [inline .travis.yml]: These are defined in the Travis environment. Currently using my credentials. Feel free to update if you like.

Comment (mdlavin) [reply .travis.yml]: Seems fine. Do you mind if I switch it to publishing as lifeomicdev?  I should have the credentials needed to do that.

Comment (mdlavin) [reply .travis.yml]: I modified the project configuration and I'll publish a new version as a test after merging this

**Pull request merged**